docs/content/props.md

```diff
@@ -58,8 +50,6 @@ const transition = {
 | `color`                   | PropTypes.string | Set CSS `color` value or `color` theme value            | `#abc123` or `primary`   |
 | `bg` or `backgroundColor` | PropTypes.string | Set CSS `background-color` value or `color` theme value | `#abc123` or `secondary` |
 
-<a name="space"></a>
-
 ## Space
 
 **Space** props used by [`Box`](/docs/api-reference#box), [`FlexBox`](/docs/api-reference#flex-box), [`Grid`](/docs/api-reference#grid), [`CodeSpan`](/docs/api-reference#code-span), [`Text`](/docs/api-reference#text), [`Link`](/docs/api-reference#link), [`Heading`](/docs/api-reference#heading), [`Quote`](/docs/api-reference#quote), [`Table`](/docs/api-reference#table), [`TableHeader`](/docs/api-reference#table-header), [`TableBody`](/docs/api-reference#table-body), [`TableRow`](/docs/api-reference#table-row), [`TableCell`](/docs/api-reference#table-cell), [`UnorderedList`](/docs/api-reference#unordered-list), [`OrderedList`](/docs/api-reference#ordered-list), and [`ListItem`](/docs/api-reference#list-item).
```

Comment:
These links are **known** not to work locally for the time being. See #877 for more thorough tracking of this issue.